docs/whats-new/staging-release-notes.md

```diff
@@ -1,8 +1,23 @@
 # Staging release notes
 
-## Aug 2023
+## September 2023
 
-**16 Aug 2023**
+**13 September 2023**
+
+Frontend version: 1.0.0-20230911.0925 | Backend version: 1.69.0-20230908.1609
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Improved tenant user management UI | We have introduced a new user management interface. This interface now enables tenants to utilise the search function and filter based on account type and user status. To provide users with flexibility, a feature flag has been introduced to toggle between this user management UI and the existing vendors UI. In the backend, modifications have been made to ensure support for filtering based on account types.|
```

Comment:
I think there's no need to mention:

> To provide users with flexibility, a feature flag has been introduced to toggle between this user management UI and the existing vendors UI.  In the backend, modifications have been made to ensure support for filtering based on account types.

Must focus on the value of this feature to the users. Does " In the backend, modifications have been made to ensure support for filtering based on account types" mean the Automation APIs has been updated?

Reply:
yeah wasnt sure if i should put. will remove.

the later was based on this description on jira: Work on the backend as well to ensure filtering based on the account types are well supported with a combination of filtering based on namespace

@JLeeChan